shlr/spp/spp.h

```diff
@@ -12,7 +12,7 @@
 int r_sys_setenv(const char *key, const char *value);
 #endif
 
-#ifdef __WINDOWS__
+#if defined (_MSC_VER) || (__WINDOWS__)
```

Comment:
imho _MSC_VER should be implicit by __WINDOWS__ msvc cant compile for any other platform and i dont think it will ever do